JWBestPracticeApps/AdvancedPlayer/AdvancedPlayer/CustomControls.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  CustomControls.swift
+//  AdvancedPlayer
+//
+//  Created by Michael Salvador on 1/25/22.
+//
+
+import UIKit
+
+protocol CustomControlsDelegate: AnyObject  {
```

Comment:
This file needs to be fully commented, so developers understand what this code's purpose is and how it relates to the rest of the application.